src/app/services/api.service.ts

```diff
@@ -83,4 +85,11 @@ export class ApiService {
   private getCsrf() {
     return this.get('csrf').map(response => response.csrf_token);
   }
+
+  private getErrorMessage(error: any): Observable<any> {
+    return error
+      ? Observable.throw(error)
+      : this.translate.get('service.api.server-error')
+          .flatMap(message => Observable.throw(message));
```

Comment:
For this to work properly, it must be modified to `.flatMap(message => Observable.throw(new Error(message)));`

Reply:
Fixed